### ISSUE TITLE
Default ASN is Not in Private Range

### ISSUE THREAD

**Issue by gmcquillan** [projectcalico/libcalico#140]
It's using 64511, where the private range begins at 64512 [1].

Value as seen in Etcd:

```
ip-10-139-4-50 ~ # etcdctl get /calico/bgp/v1/global/as_num
64511
```

From the RFC:

```
IANA has reserved, for Private Use, a contiguous block of 1023
Autonomous System numbers from the "16-bit Autonomous System Numbers"
registry, namely 64512 - 65534 inclusive.
```

Where this default value lies: 

https://github.com/projectcalico/libcalico/blob/d9fe4aa49dd193c313b481c2e5a138601e3b8b6e/calico_containers/pycalico/datastore.py#L95

1: https://tools.ietf.org/html/rfc6996

**Comment by fasaxc** [comment]
It's not quite that bad: 64511 is in the "Reserved for use in documentation and sample code" block so it's not public either:

http://www.iana.org/assignments/as-numbers/as-numbers.xhtml

That said, I agree that we shouldn't be using an AS from that block since that's not what it's for.

**Comment by gmcquillan** [comment]
Cool. I half expected that I was reading the RFC wrong. Cheers!

Thanks Shaun

**Comment by caseydavenport** [comment]
This should be fixed in the latest beta, and will be included in Calico v2.0.0
